pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -154,29 +154,30 @@ func initializePipelineRunControllerAssets(t *testing.T, d test.Data, opts pipel
 	}, cancel
 }
 
-// getTaskRunCreations will look through a set of actions to find all task run creation actions and return the set of
-// them. It will fatal the test if none are found, or if less actions were found than expected.
-func getTaskRunCreations(t *testing.T, actions []ktesting.Action, minActionCount int) []*v1beta1.TaskRun {
+// getTaskRunCreations will look through a set of taskruns generated by a specific pipelinerun and return the set of
+// them. It will fatal the test if none are found or an error occurred
```

Comment:
```suggestion
// getTaskRunsForPipelineRun returns the set of TaskRuns associated with the input PipelineRun.
// It will fatal the test if none are found or an error occurred.
```